test/Inputs/clang-importer-sdk/usr/include/module.map

```diff
@@ -151,3 +151,8 @@ module IncompleteTypes {
   header "IncompleteTypes.h"
   export *
 }
+
+module ObjCOptionalRequirements {
+  header "ObjCOptionalRequirements.h"
+  export *
+}
```

Comment:
Is this the preferred approach for a header that is shared across test directories?

Reply:
I don't think there's a single preferred way. For header code specific to a particular test, I tend to put the header in the corresponding test's `Inputs` directory and use `-import-objc-header %S/Inputs/Foo.h` instead of making a module for it, if modularization isn't essential to exercising the test.

Reply:
Agreed.